README.md

```diff
@@ -145,7 +145,7 @@ flutter build ipa
 ### macOS
 
 ```bash
-flutter build macos
+flutter build macOS
```

Comment:
Same here, the commands are correct already. They do not follow grammatical rules